pkg/commands/integrations/actions/list-permissions.go

```diff
@@ -39,8 +47,8 @@ func ListPermissions() *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVarP(&integrationID, "integration", "i", "", "integration id")
-	flags.StringVarP(&resourceType, "type", "t", "", "resource type")
+	flags.StringVarP(&integrationIDOrName, "integration", "i", "", "the integration id or type/name, for example: \"aws-account/My AWS integration\"")
```

Comment:
copy aligned with Sharon?

Reply:
Im going to sit with Sharon on the copy later.